### PR TITLE
KFSPTS-8357: Fixed SAE Collector .done file creation and error report.

### PR DESCRIPTION
**Pull request by cah292**
This fixes the issue where the .done file is not being created for the generated Collector file. The fix involves updating the appropriate code to return just the file name and not the full file path.

Also, as per a recent discussion with Nancy, I've updated the code so that it will not report a header-level validation error when writing the Collector data to a flat file. If more report-related logging for such errors is needed, then that will be addressed in a future user story.

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCreateCollectorFileServiceImpl.java [hunk 1]
```diff
@@ -110,26 +110,25 @@ protected int countCollectorFilesGeneratedFromSAEFilesToday() {
...
-                .append(collectorDirectoryPath)
-                .append(ConcurConstants.COLLECTOR_CONCUR_OUTPUT_FILE_NAME_PREFIX)
-                .append(saeFileNameWithoutExtension)
-                .append(GeneralLedgerConstants.BatchFileSystem.EXTENSION)
-                .toString();
+        return ConcurConstants.COLLECTOR_CONCUR_OUTPUT_FILE_NAME_PREFIX + saeFileNameWithoutExtension
+                + GeneralLedgerConstants.BatchFileSystem.EXTENSION;
+    }
+
+    protected String buildCollectorFilePath(String collectorFileName) {
```
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCreateCollectorFileServiceImpl.java]: should this function buildCollectorFileFullyQualifiedFileName?  We are using the "fullyQualified" naming convention when the file name includes the path.

Comment (cah292) [inline src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCreateCollectorFileServiceImpl.java]: Good catch, I will update the naming accordingly.

**Comment by cah292** [comment]
I've made some minor changes based on the initial feedback. This is now ready for review again.

**Comment by cah292** [comment]
retest this please

**Comment by jhulslander** [comment]
Changes look good to me

**Pull request merged**